frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -372,6 +385,7 @@ class EditEmail extends Component {
                 <textarea
                   id="your-response-text-area"
                   maxLength={MAX_RESPONSE}
+                  aria-label={LOCALIZE.emibTest.inboxPage.addEmailResponse.emailResponseTooltip}
```

Comment:
You'll need to make the screen reader read the textarea label AND the tooltip on field select. Now, it is only reading the tooltip content.

Reply:
fixed in 2f61ce3 and 7ba9971